src/BloomBrowserUI/pageChooser/js/page-chooser.js

```diff
@@ -43,6 +44,40 @@ var PageChooser = (function () {
         $("#preview").attr("src", $(this._selectedGridItem).find("img").first().attr("src"));
         this.setLocalizedText($('#previewDescriptionText'), 'TemplateBooks.PageDescription.', $(".pageDescription", this._selectedGridItem).text(), defaultCaptionText);
     }; // thumbnailClickHandler
+    // There's a bug deep in javascript that doesn't take into account the scrolling
+    // of a div element before something inside it is clicked on.  The following code
+    // detects whether the scrolling has changed since the last mouse click, and if so,
+    // searches for the item which should have matched.  For the initial bug report,
+    // see https://silbloom.myjetbrains.com/youtrack/issue/BL-2623.
+    // Note that the offset().top values returned by jquery properly take into account
+    // the scrollTop of the scrolling parent div.  Which makes me think the bug may be
+    // below the jquery level!?
+    PageChooser.prototype.findProperElement = function (clickedDiv, evt) {
+        var gridItem = $(clickedDiv).parent();
+        if (evt) {
+            var scrollingParent = $(clickedDiv).parentsUntil("div.bloomDialogContainer", "div.gridItemDisplay");
```

Comment:
There's only going to be one parent element with the class gridItemDisplay.  I debated how to access it, but this looked better than five levels (or was it six?) of .parent().
Storing the scrollingParent in a member variable might be a reasonable improvement, if I could figure out how to initialize it.